in.cinny.Cinny.yml

```
@@ -0,0 +1,60 @@
+id: in.cinny.Cinny
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: cinny
+rename-icon: cinny
+
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+
+  - name: Cinny
+    buildsystem: simple
+    build-commands:
+      - ls -a
+      - ar x cinny_*.deb
+      - rm -f cinny_*.deb
+      - tar xf data.tar.gz
+      - rm -f control.tar.gz data.tar.xz debian-binary
+      - cp -r usr/bin/cinny /app/bin
+      - mkdir -p /app/share/icons/hicolor
+      - cp -r usr/share/icons/hicolor/* /app/share/icons/hicolor
+      - rm -rf /app/share/icons/hicolor/512x512@2x
+      - install -Dm744 usr/share/applications/cinny.desktop /app/share/applications/in.cinny.Cinny.desktop
+    sources:
+      - url: https://github.com/cinnyapp/cinny-desktop/releases/download/v2.0.2/cinny_2.0.2_amd64.deb
+        sha256: d933cdf55640c441a4538900e52c9d1804c8639e8568c81b464a1b4896bd3741
+        type: file
+    cleanup:
+      - usr
+
+finish-args:
+  # Required due to being a GUI application
+  - --socket=x11
```

if this app supports wayland this should be:
```suggestion
  - --socket=fallback-x11
```